src/wp-includes/class-wp-script-modules.php

```diff
@@ -228,10 +274,16 @@ public function set_fetchpriority( string $id, string $priority ): bool {
 	 * @param array             $args     {
 	 *     Optional. An array of additional args. Default empty array.
 	 *
+	 *     @type bool                $in_footer     Whether to print the script module in the footer. Default 'false'. Optional.
```

Comment:
During testing, I'm seeing the new argument is only affecting block themes. For classic themes, they're always in the footer.

Unless this is PEBKAC, it would be good to document this here and for the wrapper function.